File: lakeSide-hotel-demo-client-main/src/model/Bill.js
class Bill {
    constructor(id = null, content = null, date = null, total = null, bookedRoom = null, user = null, status = null) {
        this.id = id;
        this.content = content;
        this.date = date;
        this.total = total;
        this.bookedRoom = bookedRoom;
        this.user = user;
        this.status = status;
    }
}

export default Bill;

File: lakeSide-hotel-demo-server-master/src/main/java/com/dailycodework/lakesidehotel/model/Bill.java
package com.dailycodework.lakesidehotel.model;

import jakarta.persistence.*;
import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

import java.math.BigDecimal;
import java.time.LocalDate;

@Entity
@Getter
@Setter
@AllArgsConstructor
@NoArgsConstructor
public class Bill {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private  Long id;
    @Column(columnDefinition = "TEXT")
    private String content;
    private LocalDate Date;
    private BigDecimal total;
    private String status;

    @ManyToOne(fetch = FetchType.LAZY)
    @JoinColumn(name = "bookedRoom_id")
    private BookedRoom bookedRoom;

    @ManyToOne(fetch = FetchType.LAZY)
    @JoinColumn(name = "user_id")
    private User user;
}
